packages/wallets/src/evm/index.ts

```diff
@@ -23,28 +23,31 @@ export type {
 
 export * from "./wallets/paper-wallet";
 // just the types
-export type { WalletOptions } from "./wallets/base";
 export { AbstractClientWallet } from "./wallets/base";
+export type { WalletOptions } from "./wallets/base";
+export * from "./wallets/blocto";
 export * from "./wallets/coinbase-wallet";
-export * from "./wallets/local-wallet";
+export * from "./wallets/ethers";
 export * from "./wallets/frame";
 export * from "./wallets/injected";
+export * from "./wallets/local-wallet";
+export * from "./wallets/magic";
 export * from "./wallets/metamask";
+export * from "./wallets/private-key";
 export * from "./wallets/rainbow-wallet";
-export * from "./wallets/wallet-connect";
-export * from "./wallets/wallet-connect-v1";
 export * from "./wallets/safe";
-export * from "./wallets/magic";
 export * from "./wallets/smart-wallet";
-export * from "./wallets/ethers";
-export * from "./wallets/private-key";
-export * from "./wallets/zerion";
 export * from "./wallets/trust";
-export * from "./wallets/blocto";
+export * from "./wallets/wallet-connect";
+export * from "./wallets/wallet-connect-v1";
+export * from "./wallets/zerion";
 
 export type { Chain } from "@thirdweb-dev/chains";
 
 // export the window ethereum util
-export { assertWindowEthereum } from "./utils/assertWindowEthereum";
 export { getInjectedMetamaskProvider } from "./connectors/metamask/getInjectedMetamaskProvider";
 export { getInjectedRainbowProvider } from "./connectors/rainbow/getInjectedRainbowProvider";
+export { assertWindowEthereum } from "./utils/assertWindowEthereum";
+
+// ThirdwebEmbeddedWalletSdk
+export * from "./implementations/embedded-wallet";
```

Comment:
Why break the format for wallet/connector pair and add this new "implementations" folder?